mail/templates/applicant_letter/body.html

```diff
@@ -14,8 +15,8 @@
                   <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
                       <p style="margin: 0 0 10px;">
                         Sincerely,<br />
-                        <img src="{{ base_url }}/static/images/signature-mariah-rawding.png" alt="Signature for Mariah Rawding" /><br />
-                        Mariah Rawding<br />
+                        <img src="{% if signatory.image %}{% image_version_url signatory.image "fill-300x300" %}{% else %}{% static "images/signature-mariah-rawding.png" %}{% endif %}" alt="Signature for {% if signatory.name %}{{ signatory.name }}{% else %}Mariah Rawding{% endif %}" /><br />
```

Comment:
We still need `{{ base_url }}` since the user will be viewing this outside of our website, so relative URLs won't work.